packages/mobile/src/web3/saga.test.ts

```diff
@@ -92,6 +100,34 @@ describe(getOrCreateAccount, () => {
   )
 })
 
+describe('Address getters', () => {
```

Comment:
None of these tests ever reach the `take` effect in `getAddress` right? Maybe worth having a test that uses [`dispatch`](https://redux-saga-test-plan.jeremyfairbank.com/integration-testing/dispatching.html) to mock placing the value (and improving coverage?). Huge Redux noob here, so excuse me if I'm getting something wrong!

Reply:
seems reasonable, thanks for the rec